ui/pages/_manufacturerKey/_fixtureKey.vue

```diff
@@ -135,8 +135,9 @@
       <h2>Something wrong with this fixture definition?</h2>
       <p>It does not work in your lighting software or you see another problem? Then please help correct it!</p>
       <div class="grid-3 list">
-        <a href="https://github.com/OpenLightingProject/open-fixture-library/issues?q=is%3Aopen+is%3Aissue+label%3Atype-bug" rel="nofollow" class="card"><app-svg name="bug" class="left" /><span>Report issue on GitHub</span></a>
-        <a href="/about#contact" class="card"><app-svg name="email" class="left" /><span>Contact</span></a>
+        <a href="#" class="card only-js" @click.prevent="helpWantedContext = fixture"><app-svg name="comment-alert" class="left" /><span>Send information</span></a>
+        <a href="https://github.com/OpenLightingProject/open-fixture-library/issues?q=is%3Aopen+is%3Aissue+label%3Atype-bug" rel="nofollow" class="card"><app-svg name="bug" class="left" /><span>Create issue on GitHub</span></a>
```

Comment:
Should the other two be visible when JavaScript is enabled? I'd rather have them only as a fallback.

Reply:
Having all three links at this single place should be okay.